private/protocol/jsonrpc/jsonrpc.go

```diff
@@ -16,17 +14,26 @@ import (
 
 var emptyJSON = []byte("{}")
 
-// BuildHandler is a named request handler for building jsonrpc protocol requests
-var BuildHandler = request.NamedHandler{Name: "awssdk.jsonrpc.Build", Fn: Build}
-
-// UnmarshalHandler is a named request handler for unmarshaling jsonrpc protocol requests
-var UnmarshalHandler = request.NamedHandler{Name: "awssdk.jsonrpc.Unmarshal", Fn: Unmarshal}
+// BuildHandler is a named request handler for building jsonrpc protocol
+// requests
+var BuildHandler = request.NamedHandler{
+	Name: "awssdk.jsonrpc.Build",
```

Comment:
Nit: Names can be exported constants so as to make it easier to support named handler swaps.

Reply:
For these handlers we've relied the package global handler value's `Name` parameter to do the swap.  This is how they are referred ti in the existing customization/ generated code when a swap is needed.

I can add a constant for the generic protocol error type unmarshaler since it is a constructed value, and isn't specific to any one of the protocols.

Reply:
> we've relied the package global handler value's Name parameter to do the swap. 

Not sure what you mean by this?